src/jv.h

```diff
@@ -259,6 +263,9 @@ int jv_cmp(jv, jv);
 jv jv_group(jv, jv);
 jv jv_sort(jv, jv);
 
+#ifdef __cplusplus
+}
+#endif
 
 #endif
 
```

Comment:
Unrelated to this PR: i wonder if the comment below should be cleaned up?

Reply:
Yeah, maybe!

Reply:
In addition to that, would yall be open to some light comments on what these methods do for these two files?